sdk/eventgrid/azure-messaging-eventgrid-namespaces/assets.json

```diff
@@ -1,6 +1,6 @@
 {
-  "AssetsRepo": "Azure/azure-sdk-assets",
-  "AssetsRepoPrefixPath": "java",
-  "TagPrefix": "java/eventgrid/azure-messaging-eventgrid-namespaces",
-  "Tag": "java/eventgrid/azure-messaging-eventgrid-namespaces_0ec3de5f92"
-}
+  "AssetsRepo" : "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath" : "java",
+  "TagPrefix" : "java/eventgrid/azure-messaging-eventgrid-namespaces",
+  "Tag" : ""
```

Comment:
missing tag value

Reply:
Yeah, this is very in-progress. Will get it cleaned up and re-tag you.